packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -62,6 +73,10 @@ export = {
           match: {
             serviceName: 'test.svc.local',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(10),
+            perRequest: cdk.Duration.seconds(11),
```

Comment:
Can you use different values here, instead of always 10 and 11? Will make it easier to match the assertions below.

Reply:
sure sounds good